scripts/docker-serve-full.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+# navigate to the repository root
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/.."
+
+JEKYLL_DOCKER_IMAGE_VERSION=pages-v228-minimal
+JEKYLL_DOCKER_CONTAINER_NAME=duckdb-jekyll
+
+docker run \
+    --rm \
+    --interactive \
+    --name ${JEKYLL_DOCKER_CONTAINER_NAME} \
+    --volume="${PWD}:/srv/jekyll:Z" \
+    --publish 4000:4000 \
+    --publish 35729:35729 \
+    rockstorm/jekyll:${JEKYLL_DOCKER_IMAGE_VERSION} \
+    /bin/bash -c \
+        "apk add --update gcc g++ make ruby-bundler && \
+         bundle install && \
```

Comment:
Maybe these should be in a Dockerfile so they don't have to be installed and built every time? And the image can be reused